client/my-sites/upgrades/cart/cart-messages-mixin.jsx

```diff
@@ -33,6 +32,17 @@ module.exports = {
 		);
 	},
 
+	_getBlockedPurchaseErrorMessage: function() {
```

Comment:
This is minor, we updated coding guidelines. We should no longer use `_` in React method names.